google/resource_container_node_pool_test.go

```diff
@@ -550,3 +592,25 @@ resource "google_container_node_pool" "np_with_node_config_scope_alias" {
 		oauth_scopes = ["compute-rw", "storage-ro", "logging-write", "monitoring"]
 	}
 }`, acctest.RandString(10), acctest.RandString(10))
+
+var testAccContainerNodePool_withManagement = fmt.Sprintf(`
+resource "google_container_cluster" "cluster" {
+	name = "tf-cluster-nodepool-test-%s"
```

Comment:
I think you intended on this test checking update, but since the name is changing from the basic test, it'll actually delete and recreate the cluster/np

Reply:
I didn't actually intend to test updates, but it totally _should_.  I'll change it, re-test, and circle back shortly.